intern/service/mesh/service.py

```diff
@@ -160,4 +149,16 @@ def obj_mesh(self):
             (): Returns mesh obj format
 
         """
-        return self._mesh.to_obj()
\ No newline at end of file
+        return self._mesh.to_obj()
+
+class VoxelUnits(IntEnum):
+    """Enum with valid VoxelUnits
+    """
```

Comment:
this set of lines where every line is one longer than its predecessor is the most soothing thing i've ever seen

Reply:
👯